src/plugins/intel_cpu/src/cpu_streams_calculation.cpp

```diff
@@ -17,6 +17,23 @@ using namespace InferenceEngine;
 namespace ov {
 namespace intel_cpu {
 
+std::vector<std::vector<int>> apply_hyper_threading(bool input_value,
```

Comment:
Same as for `apply_core_types`: looks like common functionality and should be moved into dev api

Reply:
updated